pkg/relayer/session/session_test.go

```diff
@@ -73,9 +73,10 @@ func requireProofCountEqualsExpectedValueFromProofParams(t *testing.T, proofPara
 	}
```

Comment:
I feel like we're very light with the tests.

Did you test this in any way?
- Unit?
- Integration?
- Manual?

Reply:
You're right.

I'll add tests.